<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QLabelImage" name="myLabel">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>5000</width>
     <height>5000</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pause_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>494</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>291</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_x"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_y"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>H</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_H"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>S</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_S"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_V"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Visualizar Imagem</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_modeView">
              <item>
               <property name="text">
                <string>Real</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cor Atual</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Segmentada</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Identificação</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Cor Atual</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_CurrentColor">
              <item>
               <property name="text">
                <string>Laranja</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Azul</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Amarelo</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Verde</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Ciano</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>H</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="Hmax_horizontalSlider">
                  <property name="maximum">
                   <number>179</number>
                  </property>
                  <property name="value">
                   <number>179</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="Hmax_spinBox">
                  <property name="maximum">
                   <number>179</number>
                  </property>
                  <property name="value">
                   <number>179</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Min</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="Hmin_horizontalSlider">
                  <property name="maximum">
                   <number>179</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="Hmin_spinBox">
                  <property name="maximum">
                   <number>179</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>S</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="Smax_horizontalSlider">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="Smax_spinBox">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_18">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Min</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="Smin_horizontalSlider">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="Smin_spinBox">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>V</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="Vmax_horizontalSlider">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="Vmax_spinBox">
                  <property name="maximum">
                   <number>255</number>
                  </property>
                  <property name="value">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="Vmin_horizontalSlider">
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="Vmin_spinBox">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="finish_pushButton">
              <property name="text">
               <string>Finish</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>376</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QPushButton" name="back_pushButton">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>100</y>
          <width>141</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>Estatísticas aqui</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLabelImage</class>
   <extends>QLabel</extends>
   <header>qlabelimage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
